src/net_info.cpp

```diff
@@ -202,12 +202,13 @@ tl::expected<net_info::RouteInfo, std::string> net_info::get_route_info(
 
 	auto iface = UNWRAP_OR_RETURN(get_if_to_use(if_name, default_interface));
 
-	if (!routes_map.contains(iface))
-		return tl::unexpected(
-		    fmt::format("Cannot find routes for interface {}", default_interface->c_str()));
-
 	close(sock);
 
+	if (!routes_map.contains(iface)) {
+		spdlog::warn("Cannot get routes for {}, returning empty RouteInfo", iface);
+		return net_info::RouteInfo{.if_name = iface};
```

Comment:
What bug does this fix?

Reply:
Can't run the tool on a bare NIC for testing with AF_XDP otherwise. Needs to be connected to a network interface

Reply:
But then you need to provide some extra details right? Like gateway mac and source ip